content/post/053-great-calamity-circumvention-assembly-at-ipfs-camp.md

```diff
@@ -0,0 +1,71 @@
+| [Building Dynamic P2P Applications](https://github.com/ipfs/camp/blob/master/DEEP_DIVES/45-building-dynamic-p2p-applications.md) | <a href="https://github.com/jkarni"><img src="https://github.com/jkarni.png?size=200" alt="jkarni" width="100" /></a> |  IPFS and its associated technologies are mostly static. Radicle is an attempt to build a dynamic P2P web on top of IPFS; one which can host applications with the same abilities of the centralized servers of today to receive inputs from various people or sources. | [Video](https://youtu.be/yboQAMwvtVs) |
+| [Better file chunking 📦](https://github.com/ipfs/camp/blob/master/DEEP_DIVES/46-better-file-chunking.md) | <a href="https://github.com/mib-kd743naq"><img src="https://github.com/mib-kd743naq.png?size=200" alt="mib-kd743naq" width="100" /></a> | Within the IPFS stack/ecosystem, just as within computing as a whole, an uncompressed stream of untagged octets is a fundamental unit of exchange. As a general-purpose data storage system IPFS needs to handle an unbounded variety of content represented by such streams. Handling the maximum amount of this variety efficiently ( ideally by default ) would likely have an outsized impact on the future adoption of IPFS as a long-term data interchange medium/format. | [Video](https://youtu.be/_cw4jx2jJrY) |
+| [Dynamic DAG traversal](https://github.com/ipfs/camp/blob/master/DEEP_DIVES/47-dynamic-dag-traversal.md) | <a href="https://github.com/maparent"><img src="https://github.com/maparent.png?size=200" alt="maparent" width="100" /></a> | Extend IPLD to cross between immutable and mutable references | [Video](https://youtu.be/W3Kz3QSVRlY) |
+| [Design Patterns for DWeb Apps](https://github.com/ipfs/camp/blob/master/DEEP_DIVES/48-design-patterns-for-dweb-apps.md) | <a href="https://github.com/leshokunin"><img src="https://github.com/leshokunin.png?size=200" alt="leshokunin" width="100" /></a> | Most IPFS apps have UX issues. We will pick a feature of an existing app, and improve its UX by simplifying how to interact with it, how it works. Textile.io lowered the permission and roles system from ~15 methods to 2 this way! | [Video](https://youtu.be/w07JvGVeHpM) |
+| [IPFS and Guix 📦](https://github.com/ipfs/camp/blob/master/DEEP_DIVES/49-ipfs-and-guix.md) | <a href="https://github.com/ambrevar"><img src="https://github.com/ambrevar.png?size=200" alt="ambrevar" width="100" /></a> | Exploration into existing or potential issues with integrating IPFS with Guix | [Video](https://youtu.be/ZvJYYZ1aEDU) |
+
+## There are more to come, keep posted!
+
+More content from IPFS Camp is being salvaged and transcoded.
+
+If you are interested in receiving an update when the next batch of videos are available, you can do this in one of the following ways:
+
+- Watch and Star the [ipfs/camp repo](https://github.com/ipfs/camp/blob/master/README.md), where all the content will live ad-eternum.
```

Comment:
technically, the github camp repo is a centralized service also susceptible to the Global Electromagnetic Fire Disaster of 2027 - maybe worth saying they will live there in addition to more permanent backup locations?